### PR TITLE
eclass/java-utils-2.eclass: drop gnu-classpath

### PR DESCRIPTION
**Pull request by vaukai**
Bug: https://bugs.gentoo.org/786093

### CODE REVIEW

**Comment by fordfrog** [comment]
there is still `dev-java/jamvm` depending on `gnu-classpath`. and the eclass also needs removal of `gnu-classpath` reference later on in the eclass.

File: eclass/java-utils-2.eclass [hunk 1]
```diff
@@ -1368,34 +1353,6 @@ java-pkg_register-environment-variable() {
...
 }
 
-# @FUNCTION: java-pkg_get-bootclasspath
-# @USAGE: <version>
-# @DESCRIPTION:
-# Returns classpath of a given bootclasspath-providing package version.
-#
-# @param $1 - the version of bootclasspath (e.g. 1.5), 'auto' for bootclasspath
-#             of the current JDK
-java-pkg_get-bootclasspath() {
```
Comment (fordfrog) [inline eclass/java-utils-2.eclass:1378]: this one is still called by another eclass:
```
eclass/java-ant-2.eclass:       local bcp="$(java-pkg_get-bootclasspath "${version}")"
```
i did not look at that eclass but maybe dropping just the `1.5` case here would be the right solution.

Comment (vaukai) [reply eclass/java-utils-2.eclass]: Sorry, I should have looked at it before deleting.
Done.

**Comment by thesamesam** [comment]
Did you send this eclass change to the ML yet?

**Comment by vaukai** [comment]
> Did you send this eclass change to the ML yet?

No. It's still pending review by @fordfrog

**Comment by fordfrog** [comment]
@vaukai i suggest we take this opportunity and get rid of all `1.5` and `1.7` occurences (i didn't find `1.6` in the eclass). what do you think?

**Comment by vaukai** [comment]
@fordfrog 

Further '1.5' I only found 
- https://github.com/gentoo/gentoo/blob/master/eclass/java-utils-2.eclass#L79-L82
- https://github.com/gentoo/gentoo/blob/master/eclass/java-utils-2.eclass#L1376
- https://github.com/gentoo/gentoo/blob/master/eclass/java-utils-2.eclass#L2548-L2549 

For '1.7' there seems to be only [#L2557-L2558](https://github.com/gentoo/gentoo/blob/master/eclass/java-utils-2.eclass#L2557-L2558). Here, in [#L2544-L2563](https://github.com/gentoo/gentoo/blob/master/eclass/java-utils-2.eclass#L2544-L2563) I guess also some vendors like 'ibm' or "jrockit" could be skipped, but not really sure about that :-(

Also '1.3' and '1.4' in examples/comments might be updated.

**Comment by fordfrog** [comment]
> @fordfrog
> 
> Further '1.5' I only found
> 
> * https://github.com/gentoo/gentoo/blob/master/eclass/java-utils-2.eclass#L79-L82
> * https://github.com/gentoo/gentoo/blob/master/eclass/java-utils-2.eclass#L1376
> * https://github.com/gentoo/gentoo/blob/master/eclass/java-utils-2.eclass#L2548-L2549
> 
> For '1.7' there seems to be only [#L2557-L2558](https://github.com/gentoo/gentoo/blob/master/eclass/java-utils-2.eclass#L2557-L2558).

yeah, i found the same, nothing more there it seems.

> Here, in [#L2544-L2563](https://github.com/gentoo/gentoo/blob/master/eclass/java-utils-2.eclass#L2544-L2563) I guess also some vendors like 'ibm' or "jrockit" could be skipped, but not really sure about that :-(

we don't have them anymore, so i guess we can drop them...

> Also '1.3' and '1.4' in examples/comments might be updated.

those i forgot to check, you are right, this is obsolete too...

**Comment by SoapGentoo** [comment]
@fordfrog will you merge?

**Comment by Flowdalic** [comment]
@vaukai this needs rebasing now and gentoo-dev@ ML review. If you rebase it, then I'll send it to the ML.

**Pull request closed without merging**